docs/operator-guide/artifacts-management/nexus-image-registry.md

```diff
@@ -0,0 +1,183 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Nexus Image Registry
+
+This documentation guide provides comprehensive instructions for integrating **Nexus Repository Manager** as a docker image registry with KubeRocketCI.
+
+## Installation
+
+To install Nexus in your environment, it's recommended to use the resources provided in the [Cluster Add-Ons](https://github.com/epam/edp-cluster-add-ons) repository. Here are detailed instructions for installing and managing the [Add-ons Application](../add-ons-overview.md). This approach involves installing both the Nexus repository manager and the [nexus-operator](https://github.com/epam/edp-nexus-operator). Leveraging the Cluster Add-Ons simplifies the deployment and management process, providing a streamlined method to integrate Nexus into your infrastructure.
+
+1. Navigate to Cluster Add-Ons repository and align [nexus values.yaml](https://github.com/epam/edp-cluster-add-ons/blob/main/clusters/core/addons/nexus/values.yaml) and [nexus-operator values.yaml](https://github.com/epam/edp-cluster-add-ons/blob/main/clusters/core/addons/nexus-operator/values.yaml):
```

Comment:
lets add "... files" at the end of sentence.

e.g. Navigate to Cluster Add-Ons repository and align the [nexus values.yaml](https://github.com/epam/edp-cluster-add-ons/blob/main/clusters/core/addons/nexus/values.yaml) and [nexus-operator values.yaml](https://github.com/epam/edp-cluster-add-ons/blob/main/clusters/core/addons/nexus-operator/values.yaml) files.